### PR TITLE
PFW-1434: MK3_3.12 Temp model calibration during wizard

### PR DESCRIPTION
**Pull request by 3d-gussner**
Cherry-pick from https://github.com/prusa3d/Prusa-Firmware/pull/3769

-[X] Tested with Prusa MK3S without MMU2

Missing for final:
Translations cherry-pick:
  - Translated messages 
    - [x] CS with Deepl
    - [x] DE
    - [x] ES with Deepl
    - [x] FR with Deepl
    - [x] IT with Deepl
    - [x] PL with Deepl
    - Community translators please translate these two messages asap.
    - [X] NL @vintagepc please review
    - [x] RO @Hauzman 
    - [x] HU @AttilaSVK 
    - [ ] HR  @Prime1910 
    - [x] SL @ingbrzy 
    - [ ] SV @Painkiller56 
    - [x] NO @OS-kar

### CODE REVIEW

File: lang/po/Firmware_no.po [hunk 1]
```diff
@@ -1771,6 +1771,17 @@ msgstr "Byttet"
...
+#. MSG_TM_CAL c=20 r=4
+#: ../../Firmware/ultralcd.cpp:4054
+msgid "Temp model cal. takes apporx. 12 mins."
+msgstr ""
+
+#. MSG_TM_NOT_CAL c=20 r=4
+#: ../../Firmware/Marlin_main.cpp:1527 ../../Firmware/Marlin_main.cpp:3404
+#: ../../Firmware/messages.cpp:170
+msgid "Temp model not calibrated yet."
+msgstr ""
```
Comment (OS-kar) [inline lang/po/Firmware_no.po]:
```suggestion
msgstr "Temp modell ikke kalibrert enda."
```

File: lang/po/Firmware_no.po [hunk 2]
```diff
@@ -1771,6 +1771,17 @@ msgstr "Byttet"
 msgid "THERMAL ANOMALY"
 msgstr "THERMISK ANOMALI"
 
+#. MSG_TM_CAL c=20 r=4
+#: ../../Firmware/ultralcd.cpp:4054
+msgid "Temp model cal. takes apporx. 12 mins."
+msgstr ""
```
Comment (OS-kar) [inline lang/po/Firmware_no.po]:
```suggestion
msgstr "Temp modell kal. tar omtrent 12 min."
```

**Review by OS-kar — APPROVED** [review]

File: lang/po/Firmware.pot [hunk 3]
```diff
@@ -1720,6 +1720,17 @@ msgstr ""
 msgid "THERMAL ANOMALY"
 msgstr ""
 
+#. MSG_TM_CAL c=20 r=4
+#: ../../Firmware/ultralcd.cpp:4054
+msgid "Temp model cal. takes apporx. 12 mins."
```
Comment (DRracer) [inline lang/po/Firmware.pot]: typo: "apporx"

**Comment by ingbrzy** [comment]
@DRracer to which branch I should push translation?

File: lang/po/Firmware.pot (hunk 3 above)
Comment (OS-kar) [reply lang/po/Firmware.pot]: looks like a spelling error on the english of all of the translations. not sure if it matters, don't know who's responsibilty it is to fix.

File: lang/po/Firmware_no.po [hunk 4]
```diff
@@ -1771,6 +1771,17 @@ msgstr "Byttet"
 msgid "THERMAL ANOMALY"
 msgstr "THERMISK ANOMALI"
 
+#. MSG_TM_CAL c=20 r=4
+#: ../../Firmware/ultralcd.cpp:4054
+msgid "Temp model cal. takes apporx. 12 mins."
```
Comment (OS-kar) [inline lang/po/Firmware_no.po]:
```suggestion
msgid "Temp model cal. takes approx. 12 mins."
```

**Review by OS-kar — APPROVED** [review]

**Comment by 3d-gussner** [comment]
> @DRracer to which branch I should push translation?

We try to make PRs to the main branch MK3 and then cherry-pick the commits to the MK3_3.12 branch. So please @ingbrzy make the PR to https://github.com/prusa3d/Prusa-Firmware/pull/3769 and add a comment that it is dependent on that one and should be cherry-picked to MK3_3.12.

Thanks @OS-kar and @ingbrzy for amazing fast replies. :heart_eyes:

**Review by DRracer — APPROVED** [review]

**Pull request merged**